### DISCUSSION TITLE
`opentelemetry-otlp` brings with disturbing logs from its deps like tonic, hyper and ever deps of deps like h2

### DISCUSSION THREAD

**Discussion by cxw620** [open-telemetry/opentelemetry-rust#1722] [Q&A]
Since `opentelemetry-jaeger` is deprecated and suggest using `opentelemetry-otlp`, I have to migrate to the latter one. However, opentelemetry-otlp brings with many disturbing logs from `tonic`, `hyper` and ever `h2` if set log level to debug:

![image](https://github.com/open-telemetry/opentelemetry-rust/assets/70561268/c689219a-29e7-4d45-9f7e-10c9f10c8a58)

I search docs for any possible methods disable these useless logs, though find nothing helpful. Setting filters like `hyper=off` is unacceptable since my own crate make use of reqwest and I sometimes needs verbose logs(what a mess that mixture of my own ones and `opentelemetry-otlp`'s.

Looking forward to any kindly replies showing me any possible solutions.

May be related: https://github.com/open-telemetry/opentelemetry-rust/issues/1146

**Comment by lalitb** [comment]
I believe the issue is related to #761. There is no solution as of now. Some effort done in #1330, if someone need to explore further. I plan to pick it up sometime when there is time.

**Reply by cijothomas** [reply]
https://github.com/open-telemetry/opentelemetry-rust/pull/1798
